packages/action-menu/src/ActionMenu.ts

```diff
@@ -62,6 +63,10 @@ export class ActionMenu extends ObserveSlotPresence(
         return this.slotContentIsPresent;
     }
 
+    private handleSlottableRequest = (event: SlottableRequestEvent): void => {
```

Comment:
is there any reason that we had to change this function definition to be a arrow function?

Reply:
Since it's used as the handler for `this.overlayElement.addEventListener`, it needs to be bound to `this`.